bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -252,15 +254,13 @@ void PreProcessor::onMessage<ClientPreProcessRequestMsg>(ClientPreProcessRequest
   {
     const auto &clientEntry = ongoingRequests_[clientId];
     lock_guard<mutex> lock(clientEntry->mutex);
-    if (clientEntry->reqProcessingStatePtr != nullptr) {
+    if (clientEntry->reqProcessingStatePtr) {
       const ReqId &ongoingReqSeqNum = clientEntry->reqProcessingStatePtr->getReqSeqNum();
-      if (ongoingReqSeqNum != reqSeqNum) {
-        LOG_WARN(GL,
-                 " reqSeqNum=" << reqSeqNum << " from clientId=" << clientId
-                               << " is ignored: previous client request=" << ongoingReqSeqNum << " is in process");
-        preProcessorMetrics_.preProcReqIgnored.Get().Inc();
-        return;
-      }
+      LOG_INFO(GL,
```

Comment:
LOG_DEBUG, maybe (INFO looks a bit excessive inside an onMessage() method)?

Reply:
All ClientRequest 'ignore' messages are 'INFO' in ReplicaImp.

Reply:
OK, thanks for the clarification!